src/packet_distribution.cpp

```diff
@@ -192,7 +191,7 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
         server_name = info.name;
         server_address =
             QString("%1:%2").arg(info.ip, QString::number(info.port));
-        window_title += ": " + server_name;
+        window_title = server_name;
```

Comment:
can you fix this if block real quick to remove the duplicate code? just tryna leave things better than the way we found them. thanks